Meta/Lagom/Tools/CodeGenerators/LibWeb/BindingsGenerator/IDLGenerators.cpp

```diff
@@ -2084,39 +2084,16 @@ JS_DEFINE_NATIVE_FUNCTION(@class_name@::@function.name:snakecase@)
 )~~~");
 }
 
-void generate_constructor_header(IDL::Interface const& interface, StringBuilder& builder)
+static void generate_prototype_or_global_mixin_declarations(IDL::Interface const& interface, StringBuilder& builder)
```

Comment:
This diff is an absolute nightmare to read, it's mostly moving code around, extracting "generate prototype" into a more generic "generate implementation".